pkgs/applications/misc/nhentai/default.nix

```diff
@@ -29,7 +29,10 @@ python3Packages.buildPythonApplication rec {
 
   meta = {
     homepage = "https://github.com/RicterZ/nhentai";
-    description = "nHentai is a CLI tool for downloading doujinshi from <http://nhentai.net>";
+    description = ''
+      **18+** | this software is used for adult content.
+      nHentai is a CLI tool for downloading doujinshi
+    '';
```

Comment:
I'm fine with dropping the link. We shouldn't be linking to pornographic content and could make a rule out of that.

Polluting `meta.description` like that is not appreciated. It is not like the description is trying to cover up, what the software is supposed to accomplish. And if users don't understand what it does, why are they installing it in the first place?

Reply:
I don't expect all those who are most likely to want to avoid such content would understand the meaning of "Hentai" here, similarly "for downloading doujinshi" is not explicitly adult content focused, from what i can tell it just means self published print.

I do agree that this is use of `meta.description` is not great, I'm just otherwise unsure how to approach it and leaving it without some clear disclaimer feels like a bad idea.

maybe it'd be best to make it clear in purpose? appending something along the longs of "from adult site(s)"

happy to remove the disclaimer part of this PR as the first goal in mind was to get rid of that link.